Makefile

```
@@ -18,7 +18,11 @@
 
 all: binaries
 
-CFLAGS = -std=c99 -Wall -O3 -g -D_GNU_SOURCE -DNO_LIBNUMA
+CFLAGS := -std=c99 -Wall -O3 -g -D_GNU_SOURCE -DNO_LIBNUMA -DNDEBUG=1
+
+ifdef WITH_TCPDIRECT
```

Please rename all TCPDIRECT to TCPDEVMEM.

```suggestion
ifdef WITH_TCPDEVMEM
```

TCP Direct is an internal name not possible for github. tcp-devmem, or devmem tcp, or 'Device Memory TCP'.